File: Drivo.Entities/LectureEntity.cs
namespace Drivo.Entities
{
    public class LectureEntity : EventEntity
    {
        public virtual StudentsGroupEntity? StudentsGroup { get; set; }
        public int? StudentsGroupId { get; set; }
        public virtual LecturerEntity? Lecturer { get; set; }
        public int? LecturerId { get; set; }
    }
}

File: Drivo/src/entities/LectureEntity.ts
import { EventEntity } from "./event.entity"
import { LecturerEntity } from "./LecturerEntity";
import { StudentsGroupEntity } from "./StudentsGroupEntity";

export class LectureEntity extends EventEntity {
    studentsGroup: StudentsGroupEntity | null;
    studentsGroupId: number | null;
    lecturer: LecturerEntity | null;
    lecturerId: number | null;
}